stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -506,3 +519,32 @@ class FPDF:
         level6: TitleStyle | None = None,
     ) -> None: ...
     def start_section(self, name: str, level: int = 0, strict: bool = True) -> None: ...
+    def use_font_face(self, font_face: FontFace) -> _GeneratorContextManager[None]: ...
+    def table(
+        self,
+        rows: Iterable[Incomplete] = (),
+        *,
+        align: str | Align = "CENTER",
+        borders_layout: str | TableBordersLayout = ...,
+        cell_fill_color: int | tuple[Incomplete, ...] | DeviceGray | DeviceRGB | None = None,
+        cell_fill_mode: str | TableCellFillMode = ...,
+        col_widths: int | tuple[int, ...] | None = None,
+        first_row_as_headings: bool = True,
+        headings_style: FontFace = ...,
+        line_height: int | None = None,
+        markdown=False,
+        text_align: str | Align = "JUSTIFY",
+        width: int | None = None,
+    ) -> _GeneratorContextManager[Table]: ...
+    @overload
+    def output(  # type: ignore[misc]
+        self,
+        name: Literal[""] | None = "",
+        dest: Unused = "",
+        linearize: bool = False,
+        output_producer_class: Callable[[FPDF], Incomplete] = ...,
```

Comment:
`.bufferize()` is called on the object returned by this callable:

https://github.com/PyFPDF/fpdf2/blob/60f5104e9a125d2ac0288a23960d0b1960e54329/fpdf/fpdf.py#L4755-L4756

I think it's meant to return an instance of `OutputProducer` (or an instance of a subclass of `OutputProducer`:

https://github.com/python/typeshed/blob/972958a406358cca95ccf0e9230f88878557abc5/stubs/fpdf2/fpdf/output.pyi#L183-L192

```suggestion
        output_producer_class: Callable[[FPDF], OutputProducer] = ...,
```